### PR TITLE
Adding TS typings and inline_query support.

### PR DESCRIPTION
**Pull request by Fazendaaa**
Thanks for [helping me out](https://github.com/telegraf/telegraf-session-redis/issues/13), I've added support to inline_query and the TypeScript typings that I mentioned. Also wrote a test for the new feature :)

### CODE REVIEW

**Comment by Fazendaaa** [comment]
Using it right now is broken some other things, please DO NOT ACCEPT THIS PR!!!

File: lib/session.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
 const debug = require('debug')('telegraf:session-redis')
 const redis = require('redis')
 
+const getRedisSessionKey = (ctx) => {
```
Comment (dotcypress) [inline lib/session.js]: I believe we good with default implementation, cause this change gonna break a lot of bots :)

Comment (dotcypress) [reply lib/session.js]: But we definitely need to merge provided typings!

**Comment by Fazendaaa** [comment]
I've improved :)

**Comment by dotcypress** [comment]
Unfortunately, provided implementation breaks backward compatibility.

**Comment by Fazendaaa** [comment]
Okay, so I will remove it and use it as you suggested. Just wait a little, I will make a TS PR :)

**Comment by Fazendaaa** [comment]
Sorry, need to fix a typo. Just one sec. I think that ctx, Context type in Telegraf,  must be first updated so that this could work on getSessionKey.

**Comment by Fazendaaa** [comment]
Fixed to receive Telegraf CTX.

**Pull request merged**